study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrugadaUpdateActivityBaseSettings.java

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.broadinstitute.ddp.db.dao.JdbiActivity;
+import org.broadinstitute.ddp.db.dao.JdbiActivityVersion;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dao.UserDao;
+import org.broadinstitute.ddp.db.dto.ActivityDto;
+import org.broadinstitute.ddp.db.dto.ActivityVersionDto;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.user.User;
+import org.broadinstitute.ddp.studybuilder.ActivityBuilder;
+import org.broadinstitute.ddp.util.ConfigUtil;
+import org.jdbi.v3.core.Handle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.nio.file.Path;
+
+public class BrugadaUpdateActivityBaseSettings implements CustomTask {
```

Comment:
Is the patch just to update 
"allowOndemandTrigger": true, in activity: MEDICAL_RECORD_RELEASE

We can just run the generic patch: UpdateActivityBaseSettings and that should update this allowOndemandTrigger value !

Reply:
You're right. Removed redundant files